applications/openshift/api-server/api_server_admission_control_plugin_AlwaysAdmit/rule.yml

```diff
@@ -4,6 +4,12 @@ prodtype: ocp4
 
 title: 'Disable the AlwaysAdmit Admission Control Plugin'
 
+{{% set custom_jqfilter = '[.data."{{.var_openshift_kube_apiserver_config_data_name}}" | fromjson]' %}}
+{{% set default_jqfilter = '[.data."config.yaml" | fromjson]' %}}
+{{% set custom_api_path = '/api/v1/namespaces/{{.var_openshift_kube_apiserver_namespace}}/configmaps/{{.var_openshift_kube_apiserver_config}}' %}}
```

Comment:
I am wondering if we want to set `var_openshift_kube_apiserver_namespace` as something like `var_cluster_namespace` instead since it will be reused by many other rules? 

```
custom_api_path = '/api/v1/namespaces{{.var_cluster_namespace}}configmaps/{{.var_openshift_kube_apiserver_config}}'
```

And Set default value for `var_cluster_namespace` as `/`.

Otherwise I think it looks good/

Reply:
I guess as for now, we need to have a tailored profile to keep track of the rules that have completely different endpoints like this one